packages/api/get-service-journeys.ts

```diff
@@ -0,0 +1,48 @@
+import { APIGatewayEvent, APIGatewayProxyResultV2 } from "aws-lambda";
+import { getServiceJourneys, isDataSource, ServiceJourneysQueryInput } from "./client";
+import { ClientError } from "./error";
+import { executeClient } from "./execute-client";
+
+export const main = async (event: APIGatewayEvent): Promise<APIGatewayProxyResultV2> =>
+    executeClient(event, getQueryInput, getServiceJourneys);
+
+const MAX_ADMIN_AREA_CODES = process.env.MAX_ADMIN_AREA_CODES || "5";
+
+export const getQueryInput = (event: APIGatewayEvent): ServiceJourneysQueryInput => {
+    const { pathParameters, queryStringParameters } = event;
+
+    const serviceRef = pathParameters?.serviceId;
+
+    if (!serviceRef) {
+        throw new ClientError("Service Ref must be provided");
+    }
+
+    const page = Number(queryStringParameters?.page ?? "1");
+
+    if (isNaN(page)) {
+        throw new ClientError("Provided page is not valid");
+    }
+
+    const dataSourceInput = queryStringParameters?.dataSource || "bods";
+
+    if (!isDataSource(dataSourceInput)) {
+        throw new ClientError("Invalid datasource provided");
+    }
+
+    const adminAreaCodes = pathParameters?.adminAreaCodes ?? "";
+    const adminAreaCodeArray = adminAreaCodes
+        .split(",")
+        .filter((adminAreaCode) => adminAreaCode)
+        .map((adminAreaCode) => adminAreaCode.trim());
+
+    if (adminAreaCodeArray.length > Number(MAX_ADMIN_AREA_CODES)) {
+        throw new ClientError(`Only up to ${MAX_ADMIN_AREA_CODES} administrative area codes can be provided`);
+    }
```

Comment:
Let's remove references to adminAreaCodes in this file and in the db query as I don't think it's necessary for the consequence page

Reply:
done